spec/assert.js

```diff
@@ -7,4 +7,24 @@ var assert = {
       return false;
     }
   }
+  isTruthy: function(a) {
+    if (a){
+      console.log("Test passed: " + a + " is truthy")
+      return true
+    }
+    else {
+      return false
+      throw new Error(a + "is not true")
+    }
+  }
+  isFalsey: function(a) {
```

Comment:
Can we rename isFalsey to isFalsy?
See: https://english.stackexchange.com/questions/109996/is-it-falsy-or-falsey